docs/getting-started-guides/ubuntu/rancher.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Rancher Integration with Ubuntu Kubernetes
+---
+
+{% capture overview %}
+This repository explains how to deploy Rancher 2.0alpha on Canonical Kubernetes. 
+
+These steps are currently in alpha/testing phase and will most likely change. 
+
+The original documentation for this integration can be found here [https://github.com/CalvinHartwell/canonical-kubernetes-rancher/](https://github.com/CalvinHartwell/canonical-kubernetes-rancher/). 
+
+{% endcapture %}
+{% capture prerequisites %}
+This page assumes you have a working Juju deployed Canonical Kubernetes Cluster. 
+{% endcapture %}
+
+
+{% capture steps %}
+## Implementation
+
+To deploy Rancher, we just need to run the Rancher container workload on-top of Kubernetes. Rancher provides their containers through dockerhub ([https://hub.docker.com/r/rancher/server/tags/](https://hub.docker.com/r/rancher/server/tags/)) and can be downloaded freely from the internet. If you're running your own registry or have an offline deployment, the container should be downloaded and pushed to the private registry.  
+
+## Deploying Rancher
+
+To deploy Rancher, we just need to run the Rancher container workload on-top of Kubernetes. Rancher provides their containers through dockerhub ([https://hub.docker.com/r/rancher/server/tags/](https://hub.docker.com/r/rancher/server/tags/)) and can be downloaded freely from the internet. If you're running your own registry or have an offline deployment, the container should be downloaded and pushed to the private registry.  
```

Comment:
This whole paragraph is duplicated, you really only need it once - either here or in the prereq. section

Reply:
Fixed, my bad.